src/linker/pipeline.py

```diff
@@ -176,15 +179,27 @@ def write_spark_module(self) -> None:
 use rule terminate_spark from spark_cluster with:
     input: rules.all.input.final_output"""
             if self.config.computing_environment == "slurm":
-                for rule in ["start_spark_master", "start_spark_worker"]:
-                    module += f"""
-use rule {rule} from spark_cluster with:
+                module += f"""
+use rule start_spark_master from spark_cluster with:
+    resources:
+        slurm_account={self.config.slurm_resources['slurm_account']},
```

Comment:
I'm hesitant to suggest DRYing this up b/c snakefiles are difficult enough to read but if there's an obvious/easy way to do it?

Reply:
I decided against looping over the rules for example because we have to overwrite the "params" as well as the resources and I was worried about decoupling that from when the resources get specified here. The 'good' way to fix all this would be to stick more of this in the 'snakemake' config dictionary (as opposed to the linker config). I will file a ticket for that refactor